tests/test_to_yaml.py

```diff
@@ -168,3 +168,14 @@ def test_structured_configs(user: User) -> None:
                 """
     )
     assert OmegaConf.to_yaml(user) == expected
+
+
+def test_raises_ValueError_on_string_input() -> None:
+    with pytest.raises(
+        ValueError,
+        match=re.escape(
+            "Invalid input. Supports one of "
+            + "[dict,list,DictConfig,ListConfig,dataclass,dataclass instance,attr class,attr class instance]"
+        ),
+    ):
+        OmegaConf.to_yaml("abc")
```

Comment:
The current fix should also catch similar issues with other placer we can to_content (e.g. OmegaConf.merge).
I suggest we test it directly instead of through another methods.

Reply:
Sounds good.

Reply:
Done in https://github.com/omry/omegaconf/pull/568/commits/61a01fc2b2b95cc21c12d7f0f9b2d162d545f300.